app/src/atoms/buttons/OnDeviceDisplay/MediumButton.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  COLORS,
+  DIRECTION_ROW,
+  Flex,
+  Icon,
+  NewPrimaryBtn,
+  SPACING,
+  styleProps,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { StyledText } from '../../text'
+import type { IconName, StyleProps } from '@opentrons/components'
+
+type MediumButtonTypes =
+  | 'primary'
+  | 'secondary'
+  | 'alert'
+  | 'alertSecondary'
+  | 'tertiary'
+  | 'tertiaryLight'
```

Comment:
Mel named `Tertiary High` so let's align as much as possible!

```suggestion
  | 'tertiaryHigh'
  | 'tertiaryHighLight'
```